src/content/1.7/modules/concepts/controllers/admin-controllers/_index.md

```diff
@@ -21,12 +21,26 @@ Somewhere in your module declare a new class that will act as a Controller:
 ```php
 // modules/your-module/src/Controller/DemoController.php
 
+namespace MyModule\Controller;
+
+use Doctrine\Common\Cache\CacheProvider;
 use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
 
 class DemoController extends FrameworkBundleAdminController
 {
+    private $cache;
+       
+    // you can use symfony DI to inject services
+    public function __construct(CacheProvider $cache)
```

Comment:
May I suggest to make 2 different examples for this ? 😄 it might be confusing for people to see the example with the methods being used (Symfony DI and fetching services from container)

I can do it if you want 😉

Reply:
done